datastew/repository/weaviate.py

```diff
@@ -58,6 +59,16 @@ def store_all(self, model_object_instances):
         for instance in model_object_instances:
             self.store(instance)
 
+    def get_all_sentence_embedders(self) -> List[str]:
+        sentence_embedders = set()
+        try:
+            result = self.client.query.get("Mapping", ["hasSentenceEmbedder"]).do()
+            for item in result['data']['Get']['Mapping']:
+                sentence_embedders.add(item["hasSentenceEmbedder"])
+        except Exception as e:
+            raise RuntimeError(f"Failed to fetch terminologies: {e}")
```

Comment:
Why is it failed to fetch terminologies here?

Reply:
I typo on my side, sorry. Fixed in the latest push.